app/views/catalog/_form_basic_info.html.haml

```diff
@@ -103,8 +103,8 @@
                             :title => _("Remove this Provisioning Entry Point"))
       - unless @edit[:new][:st_prov_type] == "generic_orchestration"
```

Comment:
This condition should hide only reconfigure for `generic_orchestration`. I have PR #3051 that removes this condition.